docs/passphrases.rst

```diff
@@ -85,3 +85,16 @@ journalist will require the following passphrases:
 The backup that is created during the installation of SecureDrop is also
 encrypted with the application's GPG key. The backup is stored on the
 persistent volume of the Admin Live USB.
+
+*Transfer Device* and *Export Device*
```

Comment:
We should probably reiterate/link to the printer suggestion here as a reminder

Reply:
Done in 2f62f1321.